src/CachePool.php

```diff
@@ -97,8 +105,6 @@ public function set($key, $value, $ttl = null)
             throw new CacheException(sprintf('Could not normalize cache TTL'));
         }
 
-        $ttl = is_null($ttl) ? 0 : $ttl;
```

Comment:
IMO this still needs to be done, because in WP there's no such thing as a `null` TTL.

Reply:
With the change to the default TTL no longer able to be `null`, the `$ttl` can't be `null` at this stage.

Reply:
True.